simple-todo/webpack.config.js

```diff
@@ -9,33 +15,29 @@ module.exports = {
     filename: 'build.js'
   },
   module: {
-    loaders: [
-      {
-        test: /\.vue$/,
-        loader: 'vue'
-      },
-      {
-        test: /\.js$/,
-        loader: 'babel',
-        exclude: /node_modules/
-      },
+    rules: [
+      { test: /\.vue$/, loader: 'vue-loader' },
+      { test: /\.css$/, loader: 'css-loader' },
+      { test: /\.scss$/, loader: 'css-loader' },
+      { test: /\.js$/, loader: 'babel-loader', exclude: /node_modules/ },
```

Comment:
babel-loader can no longer be referenced as just babel.
`npm run dev` throws an error if this is missing.